sdk/storage/storage-blob/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 12.0.3 (Unreleased)
 
+- Fixed `api-extractor` related issue [PR #3513](https://github.com/Azure/azure-sdk-for-js/pull/3513)
```

Comment:
I'd either skip the changelog entry altogether because it doesn't affect the end user much or make it more specific like 

> Exposing the interface `OperationTracingOptions` in the API surface to support tracing features

Reply:
Good point! will update

Reply:
No changes to CHANGELOG now